portfolio/src/main/webapp/index.html

```diff
@@ -2,74 +2,133 @@
         <div class = "about-info">
             <p>Hi! I am a BTech undergrad at IIIT Delhi pursuing Computer Science and Design Engineering.</p>
             <p>I recently completed my internship at MathWorks where I worked in the Simulink UI Team.</p>
             <p>I enjoy Data Structures and Algorithms, Database Management and Human-Computer Interaction.</p>
             <p>My hobbies include Karate, swimming, illustration and animation.</p>
+            <br>
+            <p>Click here to get a random fact about me!</p>
+            <br>
+            <button onclick = "addRandomFact()">PRESS</button>
+            <br>
+            <div id = "fact-container"></div>
+            </div>
         </div>
     </div>
 
     <div class = "container skills" id = "skills">
         <h1>Skills</h1>
-        <div class = "skills-info">
-            <p>Expertise Area</p>
-            <ul>
-                <li>Data Structures and Algorithms</li>
-                <li>Database Management</li>
-                <li>Human Computer Interaction</li>
-                <li>Designing Interactive Systems</li>
-            </ul>
-            <p>Programming Languages</p>
-            <ul>
-                <li>Java</li>
-                <li>Python</li>
-                <li>C++</li>
-                <li>MATLAB</li>
-            </ul>
+        <div class = "html">
+            <div class = "tag-DSA">
+                <p>DSA</p>
+            </div>
+            <div class = "progressbar">
+                <div class = "bar-DSA"></div>
+            </div>
+        </div>
+        <div class = "html">
+            <div class = "tag-dbms">
+                <p>DBMS</p>
+            </div>
+            <div class = "progressbar">
+                <div class = "bar-dbms"></div>
+            </div>
+        </div>
+        <div class = "html">
+            <div class = "tag-hci">
+                <p>HCI</p>
+            </div>
+            <div class = "progressbar">
+                <div class = "bar-hci"></div>
+            </div>
+        </div>
+        <div class = "html">
+            <div class = "tag-dis">
+                <p>UI/UX</p>
+            </div>
+            <div class = "progressbar">
+                <div class = "bar-dis"></div>
+            </div>
         </div>
     </div>
 
     <div class = "container work" id = "work">
         <h1>Work</h1>
+        <p>Previously interned at:</p>
         <div class = "work-info">
             <ol>
-                <li>MathWorks</li>
-                <li>CRIS</li>
-                <li>CodeChef</li>
+                <li>1. MathWorks: Created a search index generator for a set of Simulink Data Model instances using metamodel reflection</li>
+                <li>2. CRIS: Co-developed the CM2S2 App for CMM department using Android Studio</li>
+                <li>3. CodeChef: Created, formulated and modified lesson plans on logical reasoning, problem solving and python programming and executed them.</li>
```

Comment:
suggestions: If possibly you can add hyperlinks to the work here. This need not be done now.